utils/data_version_control.py

```diff
@@ -98,3 +102,63 @@ def version_new_data(filename_roots: List[str]) -> None:
     dvc_files = [f"{fname}.csv.dvc" for fname in filename_roots]
     add_csv_files_to_dvc(csv_files)
     add_and_commit_dvc_files_to_git(dvc_files)
+
+
+def git_checkout_folder(
```

Comment:
There needs to be tests added (happy and unhappy path) for these three functions